.github/workflows/test.yml

```
@@ -0,0 +1,56 @@
+name: Test
+
+on:
+  pull_request:
+    types: [ opened, edited, reopened, synchronize ]
```

It's better to add `ready_for_review` type, we don't need to run test on draft pull requests

```suggestion
    types: [ opened, edited, reopened, synchronize, ready_for_review ]
```